dude.py

```diff
@@ -231,8 +233,87 @@ def svn_interaction(self):
+\renewcommand\contentsname{Tagesordnung}
+\newenvironment{antrag}{\begin{quote}\begin{itshape}}{\end{itshape}\end{quote}}
+\usepackage{hyperref}
+%-------------------------------------------------
+% Konsensvorlagen (ggf. anpassen!)
+%-------------------------------------------------
+\newcommand{\konsens}[1]{In der Fachschaftssitzung MathPhysInfo, sowie in den anwesenden Fachschaftsräten, besteht Konsens ohne Bedenken.\\} % immer die Anzahl der Anwesenden anpassen!
+\newcommand{\konsensLB}[1]{In der Fachschaftssitzung MathPhysInfo, sowie in den anwesenden Fachschaftsräten, besteht Konsens mit leichten Bedenken.\\} % immer die Anzahl der Anwesenden anpassen!
+\newcommand{\konsensE}[1]{In der Fachschaftssitzung MathPhysInfo, sowie in den anwesenden Fachschaftsräten, besteht Konsens mit Enthaltung.\\} % immer die Anzahl der Anwesenden anpassen!
+\newcommand{\konsensFsrPhys}{Die Fachschaftsratssitzung Physik entscheidet einstimmig, den Beschluss entsprechend der Entscheidung der Fachschaftssitzung MathPhysInfo umzusetzen.\\}
+% \newcommand{\konsensFsrMathe}{Die Fachschaftsratssitzung Mathematik entscheidet einstimmig, den Beschluss entsprechend der Entscheidung der Fachschaftssitzung MathPhysInfo umzusetzen.\\}
+\newcommand{\konsensFsrInfo}{Die Fachschaftsratssitzung Informatik entscheidet einstimmig, den Beschluss entsprechend der Entscheidung der Fachschaftssitzung MathPhysInfo umzusetzen.\\}
+
+\setlength{\parindent}{0pt}
+\setlength{\parskip}{1em}
+
+\begin{document}
+\date{\vspace{-2em}6. November 2019\vspace{-1em}} % Datum ersetzen
+\title{\vspace{-2em}Protokoll der Fachschaftssitzung MathPhysInfo}
+\maketitle
+
+\begin{tabbing}
+    \textbf{Sitzungsmoderation:}\quad\=Kai-Uwe \\ % SiMo einfügen
+    \textbf{Protokoll:}\> Max M\"uller \\% Protokoll einfügen
+    \textbf{Beginn:}\>18:15 Uhr\\
+    \textbf{Ende:}\>xx:xx Uhr\\ % Sitzungsende einfügen
+\end{tabbing}
+
+\section{Begrüßung}
+    Die Sitzungsmoderation begrüßt die anwesenden Mitglieder der Studienfachschaften Mathematik, Physik und Informatik und eröffnet so die Fachschaftsvollversammlung der Studienfachschaften Mathematik, Physik und Informatik.
+
+\section{Feststellung der Beschlussfähigkeiten}
+    Fachschaftsrat Physik, Mathe und Informatik sind alle Beschlussfähig.
+
+\section{Beschluss des Protokolls der letzten Sitzung}
+
+\begin{antrag}
+	Annahme des Protokolls vom xx. Monat 2019.
+\end{antrag}
+\konsensE{}
+
+\section{Feststellen der Tagesordnung}
+\begin{antrag}
+    Die Tagesordnung wird in der vorliegenden Form angenommen.
+\end{antrag}
+\konsens{}
+
+\section{Sitzungsmoderation für die nächste Sitzung}
+    Die Sitzungsmoderation für die Fachschaftssitzung MathPhysInfo der nächsten Woche wird von xxx übernommen. % SiMo nachste Woche einfugen
+
+"""
+
+        for top in self.tops[5:]:
+            top.title.title_text = top.title.title_text.replace("&", "\\&")
+            einladung += "\\section{" + top.title.title_text[top.title.title_text.find(":")+2:] + "}\n\n"
```

Comment:
This index stuff is pretty unreadable, please change it or add a comment on how this works